calico_kubernetes/calico_kubernetes_cni.py

```diff
@@ -227,50 +243,17 @@ def _set_profile_on_endpoint(endpoint, profile_name):
     """
     _log.info('Configuring Pod Profile: %s' % profile_name)
 
-    if not datastore_client.profile_exists(profile_name):
+    if not _datastore_client.profile_exists(profile_name):
         _log.info("Creating new profile %s." % (profile_name))
-        datastore_client.create_profile(profile_name)
-        # _assign_default_rules(profile_name)
+        _datastore_client.create_profile(profile_name)
+        _apply_rules(container_id, namespace, pod_name, profile_name)
```

Comment:
I think `_apply_rules` is not applicable any more after https://github.com/projectcalico/libcalico/pull/9, which is in the latest release of libcalico.

It might be worth including the code to update to libcalico v0.3.0 in this PR.